libaums/src/main/java/com/github/mjdev/libaums/fs/fat32/FatDirectory.java

```diff
@@ -607,4 +607,8 @@ public void delete() throws IOException {
 		parent.write();
 		chain.setLength(0);
 	}
+
+	public boolean isEmpty(){
+		return (entries.size() == 0);
```

Comment:
You probably want to call `init()` first.

Also what does it mean for a directory to be empty? Usually each dir in FAT has current and parent directory entries. So this will always return 2 at minimum.